src/LedgerLivePlatformSDK/index.ts

```diff
@@ -125,29 +125,45 @@ export default class LedgerLivePlatformSDK {
   /**
    * Start the exchange process by generating a nonce on Ledger device
    * @param {ExchangeType} exchangeType
-   * @param {string} partnerName
    *
    * @returns {Promise<string>} The nonce of the exchange
    */
-  async initExchange(
-    _exchangeType: ExchangeType,
-    _partnerName: string
-  ): Promise<string> {
-    throw new Error("Function is not implemented yet");
+  async startExchange(_exchangeType?: ExchangeType): Promise<string> {
+    const exchangeType = { SWAP: 0x00, SELL: 0x01, FUND: 0x02 }[
+      _exchangeType || "SWAP"
+    ];
+    return this._request("exchange.start", { exchangeType });
   }
```

Comment:
Following above comment regarding use of enum, this would get us something like that:
PS: I removed the trailing `_` for the `_exchangeType` param since it was only added because the argument was not yet used (since the function was not implemented 😉)

```suggestion
  async startExchange(exchangeType?: ExchangeType): Promise<string> {
    return this._request("exchange.start", {
      exchangeType: exchangeType || ExchangeType.SWAP,
    });
  }
```

Reply:
Also, do we really want the exchange type to be an optional parameter? If yes, why? Since the `"exchange.start"` call requests an exchangeType anyways?

Reply:
That's just me being swap biased and wanting to default to swap, but you are the SDK folks and I don't have a strong opinion here